grafana-plugins/topo-plugin/.gitignore

```diff
@@ -1,30 +1,4 @@
-# Logs
-logs
-*.log
-npm-debug.log*
-yarn-debug.log*
-yarn-error.log*
-
+dist/
+yarn.lock
+coverage/
```

Comment:
I think `dist` should be removed from this file as we have decided to retain this directory at the repository. Besides, why are the original file patterns deleted, like `logs`?